cmd/cmd.go

```diff
@@ -690,6 +692,45 @@ func showInfo(resp *api.ShowResponse, w io.Writer) error {
 		})
 	}
 
+	if resp.ModelInfo != nil && verbose {
+		tableRender("Metadata", func() (rows [][]string) {
+			keys := make([]string, 0, len(resp.ModelInfo))
+			for k := range resp.ModelInfo {
+				keys = append(keys, k)
+			}
+			sort.Strings(keys)
```

Comment:
Does it make sense to sort these vs provide them as they are? I realize we might do this on ollama.com but the the order would be good to keep

Reply:
ModelInfo is a `map[string]any` inside of `api/types.go` so we'd have to change the API in order to preserve the order since hashmaps don't guarantee any kind of ordering. This was just trying to mimic ollama.com and is less invasive.